pkg/install/removebootstrap.go

```diff
@@ -26,3 +29,18 @@ func (i *Installer) removeBootstrap(ctx context.Context) error {
 	i.log.Print("removing bootstrap nic")
 	return i.interfaces.DeleteAndWait(ctx, resourceGroup, "aro-bootstrap-nic")
 }
+
+func (i *Installer) removeBootstrapIgnition(ctx context.Context) error {
+	i.log.Print("remove ignition config")
+
+	blobService, err := i.getBlobService(ctx, mgmtstorage.Permissions("d"))
+	if err != nil {
+		return err
+	}
+
+	// You cannot grant permissions to delete a container with a service SAS.
+	// TOFIX: We should be using AccountSAS for this later on
```

Comment:
This doesn't make sense to me - AccountSAS is already being used.  Are you sure that this isn't because you are requesting ResourceTypes "o" (object) and you actually need "c" (container)?